modules/csharp/examples/code_snippets/VectorSearch.cs

```diff
@@ -67,14 +66,14 @@ private void CreateCustomVectorIndexConfig()
                 MinTrainingSize = 100,
                 MaxTrainingSize = 200
             };
-            // end::vs - create - default - config[]
+            // end::vs-create-custom-config[]
         }
 
         private void CreateVectorIndex()
         {
-            var database = new Database("my-database");
-
             // tag::vs-create-index[]
+            var database = new Database("my-database");
```

Comment:
In general I want to leave lines like this out of the snippets. It should be obvious that you need to create a database, and this just adds needless lines to the snippet.